### DISCUSSION TITLE
How to perfect your python.

### DISCUSSION THREAD

**Discussion by Onesmus-dev** [codewars/codewars.com#3150] [General]
When you are a self taught programmer it is a bit challenging to handle interview questions. How can one master the craft of coding be able to twist is knowledge or earned skill to answer this question ?

**Comment by atlasaait** [comment]
Pratice a lot.

**Reply by mujahedISA** [reply]
may god bless u with the strength of 10 grown men

**Comment by atlasaait** [comment]
Assalamu'alaykum, how do you know me?

Le mar. 20 févr. 2024 à 20:42, Mujahed T Issa ***@***.***> a
écrit :
